contracts/world/sources/character/character.move

```diff
@@ -36,6 +36,7 @@ public struct Character has key {
     id: UID,
     key: TenantItemId, // The derivation key used to generate the character's object ID
     tribe_id: u32,
+    character_address: address,
```

Comment:
Am I understanding correctly that the Character is now an Owned object and not a Shared Object?

Reply:
yeah I could do those extra queries, we're already doing it for assemblies but that's because they can change ownership. Character don't, so we don't really have to. 

Having `character_address` on the `Character` struct also help storage unit creation or any future object we need to transfer to a character because we can simply pass `Character{}` to those create functions and from that they have the tenant, the address and everything they need to proceed.

Reply:
All right, SGTM.